### ISSUE TITLE
[BUG] Tabs-Bar with buttons focus behavior is wrong

### ISSUE THREAD

**Issue by anotherMatze** [porsche-design-system/porsche-design-system#2011]
Description  
---
Our Tabs Bar has a specific keyboard behavior. If we reach the Tabs Bar by pressing the `tab` key, the active tab gets focused. If we want to switch tabs, we have to use `arrow left/right` keys to cycle through the tabs. When the focus is moved away from the active tab with an arrow key right and we press tab the focus should move outside of the tabs bar, onto the next focusable element. At the moment it moves back to the active tab. (See video)

To Reproduce
---
1. Go to https://designsystem.porsche.com/v2/components/tabs-bar/examples
2. Use any example, click on tab element to get an active tab
3. Reach the active tab by pressing tab
4. press arrow right to move the focus away from the active tab
5. press tab again
6. focus is on active tab instead on next focusable element

Expected behavior
---
- Second tab stroke should move focus out of tabs bar onto the next focusable element.
- Behavior should be like described here https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html

Screenshots
---
https://user-images.githubusercontent.com/60395476/185573577-22ef635c-10e9-4156-8bc9-36c7f9f8cd77.mov

Test Environment
---
Platform/Browser/Version: [e.g. iOS Safari 11, macOS Chrome 81]
